Packs/MicrosoftGraphDeviceManagement/ReleaseNotes/1_1_20.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Microsoft Endpoint Manager (Intune)
+
+- Added a note in the README file that by default physicalMemoryInBytes will return 0 from the command ***msgraph-get-managed-device-by-id***,
```

Comment:
The README is not relevant to the RN. 
We should:
1. Add the note from the readme also to the yml file.
2. Update the RN accordingly.

Reply:
done